### DISCUSSION TITLE
Android - How should use core-wallet to signing a Tether-ERC20 transaction?

### DISCUSSION THREAD

**Discussion by chamid** [trustwallet/wallet-core#1637] [Q&A]
I explore the sample android app
And there is an example of signing a transaction using EthereumSigner
But i want to know how could i use this to sign a transaction for example to transfer Tether-ERC20???

**Comment by RobinOnTheWay** [comment]
Have you solved this? I am exploring this too. Didn't get any clue.

**Reply by chamid** [reply]
Nothing yet but I think it might be adding contract id or something else
her are some example:

[https://github.com/trustwallet/wallet-core/blob/master/android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/ethereum/TestEthereumTransactionSigner.kt](https://github.com/trustwallet/wallet-core/blob/master/android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/ethereum/TestEthereumTransactionSigner.kt)
